<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 10</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label_0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выбор цвета</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>741</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="label_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Цвет фона:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="fontColorButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Цвет поверхности:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="surfaceColorButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выбор уравнения</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>y = 0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y = sin(x)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y = x^2 * y^2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="label_20">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Границы</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>xmin:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditXmin">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_19">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>xmax:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditXmax">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>zmin:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditZmin">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_22">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>zmax:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditZmax">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QLabel" name="label_18">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Шаг</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>dx:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditDX">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_23">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>dz:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditDZ">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="4">
     <widget class="QLabel" name="label_24">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Угол относительно оси</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>ox:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QSlider" name="sliderX">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="labelOX">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_11">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>oy:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="2">
     <widget class="QSlider" name="sliderY">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QLabel" name="labelOY">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>oz:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1" colspan="2">
     <widget class="QSlider" name="sliderZ">
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QLabel" name="labelOZ">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="4">
     <widget class="QLabel" name="label_26">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Масштаб</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QLabel" name="label_27">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>По x:</string>
      </property>
     </widget>
    </item>
    <item row="18" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditXS">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="2">
     <widget class="QLabel" name="label_28">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>По y:</string>
      </property>
     </widget>
    </item>
    <item row="19" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditYS">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="4">
     <widget class="QPushButton" name="setButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Задать</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="4">
     <widget class="QPushButton" name="discardButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0" colspan="4">
     <widget class="QPushButton" name="discardAllButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </item>
    <item row="23" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="24" column="0" colspan="4">
     <widget class="QPushButton" name="aboutButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>О программе</string>
      </property>
     </widget>
    </item>
    <item row="25" column="0" colspan="4">
     <widget class="QPushButton" name="authorButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Об авторе</string>
      </property>
     </widget>
    </item>
    <item row="26" column="0" colspan="4">
     <widget class="QPushButton" name="outButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
